webpack/assets/javascripts/react_app/Root/Context/ForemanContext.js

```diff
@@ -10,6 +10,7 @@ const useForemanMetadata = () => useForemanContext().metadata;
 export const useForemanVersion = () => useForemanMetadata().version;
 export const useForemanSettings = () => useForemanMetadata().UISettings;
 export const useForemanDocUrl = () => useForemanMetadata().docUrl;
-export const useForemanOrganization = () => useForemanMetadata().organization;
-export const useForemanLocation = () => useForemanMetadata().location;
+export const useForemanOrganization = () =>
+  useForemanMetadata().organization?.title;
+export const useForemanLocation = () => useForemanMetadata().location?.title;
```

Comment:
In case someone else will want to use something other than the `title`
it would be better to return only the `org/loc` and take the needed value inside your component.